src/app/components/wrapper-components/part-creation/part-creation.component.spec.ts

```diff
@@ -292,16 +294,20 @@ describe('PartCreationComponent:', () => {
             await testUtils.clickElement('#partTypeBlitz');
             testUtils.detectChanges();
 
-            expect(component.configFormGroup.get('maximalMoveDuration').value).toBe(PartType.BLITZ_MOVE_DURATION);
-            expect(component.configFormGroup.get('totalPartDuration').value).toBe(PartType.BLITZ_PART_DURATION);
+            expect(Utils.getNonNullable(component.configFormGroup.get('maximalMoveDuration')).value)
+                .toBe(PartType.BLITZ_MOVE_DURATION);
+            expect(Utils.getNonNullable(component.configFormGroup.get('totalPartDuration')).value)
+                .toBe(PartType.BLITZ_PART_DURATION);
         }));
         it('should update the timings when reselecting normal part', fakeAsync(async() => {
             await testUtils.clickElement('#partTypeBlitz');
             await testUtils.clickElement('#partTypeStandard');
             testUtils.detectChanges();
 
-            expect(component.configFormGroup.get('maximalMoveDuration').value).toBe(PartType.NORMAL_MOVE_DURATION);
-            expect(component.configFormGroup.get('totalPartDuration').value).toBe(PartType.NORMAL_PART_DURATION);
+            expect(Utils.getNonNullable(component.configFormGroup.get('maximalMoveDuration')).value)
+                .toBe(PartType.NORMAL_MOVE_DURATION);
```

Comment:
prefer constantification to line break !

Reply:
fait